### DISCUSSION TITLE
Wrong language is downloading with video

### DISCUSSION THREAD

**Discussion by vperalta57** [aclap-dev/video-downloadhelper#1557] [Help and Q&A]
### Your question

sample video https://www.youtube.com/watch?v=kvH3vn8xoqQ&t=11s

Hello,

I use video-download-helper on Firefox. I have found of late that some of the videos I have downloaded are in foreign languages. I checked the Firefox settings and am only using English. I checked to make sure that both download-helper and Firefox were running the latest software. I tested downloading videos with and without Co App. The sample URL above plays in English but download in another language. I am at a loss of what the issue is. Can you help?

### Addon & CoApp version and details

VDH - 9.1.0.44 
CoApp - v2.0.19
OS - Windows 10 64bit
Browser Firefox 133

**Comment by jcfrog** [comment]
It's a known issue, the dev team is working on it.

**Reply by GrampaWildWilly** [reply]
There are other discussions in this forum on this topic.  You should use the GitHub search feature to find them & read them.
